src/components/notifications-table/NotificationsTable.jsx

```diff
@@ -0,0 +1,212 @@
+import i18n from "@dhis2/d2-i18n";
+import { ObjectsTable, withSnackbar, withLoading, ConfirmationDialog } from "d2-ui-components";
+import { withRouter } from "react-router-dom";
+import { withStyles } from "@material-ui/core/styles";
+
+import PageHeader from "../shared/PageHeader";
+
+import SyncReport from "../../models/syncReport";
+import SyncSummary from "../sync-summary/SyncSummary";
+import { createMetadataExportUrl } from "../../utils/synchronization";
+
+const styles = () => ({
+    tableContainer: { marginTop: -10 },
+});
+
+class NotificationsTable extends React.Component {
+    state = {
+        tableKey: Math.random(),
+        toDelete: null,
+        summaryOpen: false,
+        syncReport: SyncReport.create(),
+    };
+
+    static propTypes = {
+        d2: PropTypes.object.isRequired,
+        snackbar: PropTypes.object.isRequired,
+        history: PropTypes.object.isRequired,
+        loading: PropTypes.object.isRequired,
+    };
+
+    static model = {
+        modelValidations: {
+            href: { type: "URL" },
+            timestamp: { type: "DATE" },
+            metadata: { type: "COLLECTION" },
+        },
+    };
+
+    backHome = () => {
+        this.props.history.push("/");
+    };
+
+    deleteNotification = notification => {
+        this.setState({ toDelete: notification });
+    };
+
+    cancelDelete = () => {
+        this.setState({ toDelete: null });
+    };
+
+    confirmDelete = async () => {
+        const { toDelete } = this.state;
+
+        const syncReport = new SyncReport(toDelete);
+        this.setState({ toDelete: null });
+        this.props.loading.show(true, i18n.t("Deleting notification"));
+
+        await syncReport.remove(this.props.d2).then(response => {
+            this.props.loading.reset();
+
+            if (response.status) {
+                this.props.snackbar.success(i18n.t("Deleted {{name}}", { name: toDelete.name }));
+
+                // TODO: Workaround, add a way to force render of ObjectsTable on-demand
+                this.setState({ tableKey: Math.random() });
+            } else {
+                this.props.snackbar.error(
+                    i18n.t("Failed to delete {{name}}", { name: toDelete.name })
+                );
+            }
+        });
+    };
+
+    openSummary = data => {
+        this.setState({ summaryOpen: true, syncReport: new SyncReport(data) });
+    };
+
+    closeSummary = () => {
+        this.setState({ summaryOpen: false });
+    };
+
+    // TODO: We should fix d2-ui-components instead
+    getValueForCollection = values => {
+        const namesToDisplay = _(values)
+            .map(value => value.displayName || value.name || value.id)
+            .compact()
+            .value();
+
+        return (
+            <ul>
+                {namesToDisplay.map(name => (
+                    <li key={name}>{name}</li>
+                ))}
+            </ul>
+        );
+    };
+
+    // TODO: We should fix d2-ui-components instead
```

Comment:
We'll see this on the meeting, bypassing the formatter was a way to get full control of how a field will be rendered.